conf/core/ConfigOptions.asciidoc

```diff
@@ -1304,6 +1318,13 @@ down debugging significantly, so use with caution.
 If false, when running Differential Conflation the reference elements will not be dropped from the
 output. Setting this to false is primarily useful when debugging unconnected way snapping issues.
 
+=== differential.remove.river.partial.matches.as.whole
+
```

Comment:
I haven't tested the partial removal a lot with rivers, but I saw enough datasets that conflated super slowly with it on to decide to disable by default for rivers only. The slow runtimes will occur with lengthy rivers only.